### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Review by alexzubkoff — APPROVED** [review]
Looks good! Nothing to add...

File: taxi/models.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return f"{self.username} ({self.first_name} {self.last_name}) " \
+               f"license number: {self.license_number}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=70)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="car_manufacturer",
```
Comment (Y-Havryliv) [inline taxi/models.py:35]: remind yourself how to name `related_name` properly
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency

File: taxi/models.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    )
+    drivers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name="drivers",
```
Comment (Y-Havryliv) [inline taxi/models.py:39]: same here

File: taxi_service/settings.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    }
+}
+
+
+# Password validation
+# https://docs.djangoproject.com/en/4.1/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
```
Comment (Y-Havryliv) [inline taxi_service/settings.py:90]: fix this

File: venv/pyvenv.cfg [hunk 4]
```diff
@@ -0,0 +1,8 @@
+home = C:\Users\Best\AppData\Local\Programs\Python\Python310
+implementation = CPython
```
Comment (Y-Havryliv) [inline venv/pyvenv.cfg:2]: why did you push it?

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: venv/pyvenv.cfg (hunk 4 above)
Comment (Vasyl-Poremchuk) [reply venv/pyvenv.cfg]: I didn't notice why I added this file)

**Review by Y-Havryliv — APPROVED** [review]
Nice!

**Pull request closed without merging**